.github/workflows/update_golang_version.yml

```diff
@@ -0,0 +1,74 @@
+name: Update Golang Version
+
+on:
+  schedule:
+    - cron: "0 0 * * *" # Runs every day at midnight UTC
+  workflow_dispatch:
+
+jobs:
+  update_golang_version:
+    strategy:
+      matrix:
+        branch: [ main, release-16.0, release-15.0, release-14.0 ]
+    name: Update Golang Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.20.2
+
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ matrix.branch }}
+
+      - name: Detect new version and update codebase
+        id: detect-and-update
+        run: |
+          old_go_version=$(go run ./go/tools/go-upgrade/go_upgrade.go get_go_version)
+          echo "old-go-version=${old_go_version}" >> $GITHUB_OUTPUT
+
+          if [ ${{ matrix.branch }} == "main" ]; then
+            go run ./go/tools/go-upgrade/go_upgrade.go --no-workflow-update --main --allow-major-upgrade
+          else
+            go run ./go/tools/go-upgrade/go_upgrade.go --no-workflow-update
+          fi
+
+          output=$(git status -s)
+          if [ -z "${output}" ]; then
+            exit 0
+          fi
+          echo "create-pr=true" >> $GITHUB_OUTPUT
+
+          go_version=$(go run ./go/tools/go-upgrade/go_upgrade.go get_go_version)
+          bootstrap_version=$(go run ./go/tools/go-upgrade/go_upgrade.go get_bootstrap_version)
```

Comment:
I think this should now be:
```
bootstrap_version=$(go run ./go/tools/go-upgrade/go-upgrade.go get bootstrap_version)
```

Similar for the other call spots in the workflow file.

Reply:
fixed via a5a4327acc